### PR TITLE
pda clears alert icon when replying from chat

### PR DESCRIPTION
**Pull request by etherware-novice**
## What this does
when clicking reply to a pda message from the chat box, the alert icon on the pda is cleared

## Why it's good
makes the alert icon more accurately represent "unread" notifications, also helps if you do miss a pda message in the chat logs

## Changelog

:cl:
 * tweak: replying to pda messages from chat clears alert icon

### CODE REVIEW

**Comment by etherware-novice** [comment]
who

**Comment by DeityLink** [comment]
Oh thank god this one was my pet peeve for years but I always forget about it when not in-game.

@TheLastAussie based on what? This PR is a two line bugfix that doesn't seem to break anything.

@etherware-novice a duo of coders who were kinda infamous for not adequately testing their PRs. Ada in particular broke a lot of stuff in the game.

**Comment by nervere** [comment]
> This guy is the next Adam/Kanef I can sense it in my balls

Completely braindead to comment on and downvote a fix based on some made up nonsense. Next time, don't comment at all.

File: code/game/objects/items/devices/PDA/apps/messenger.dm [hunk 1]
```diff
@@ -89,6 +89,8 @@
                 U << browse(null, "window=pda")
                 return
         if("Message")
+            overlays.len = 0  // replying to a message from chat clears alert
+            update_icon()
```
Comment (DeityLink) [inline code/game/objects/items/devices/PDA/apps/messenger.dm]: You're in the proc of the messenger app's datum. You need to do P.overlays.len = 0 and P.update_icon() instead (and obviously move those lines below the one that defines P).

**Comment by DeityLink** [comment]
After ada I expect we'll be more regarding toward the quality of PRs, but regardless no need to be so antagonistic. People can improve. Alternatively you could offer more context on why they got banned instead of just dropping that info.

@etherware-novice that being said while ada was often criticized for not testing their code, you should at least try to compile yours.

**Pull request merged**